app/scripts/lib/app-start.js

```diff
@@ -726,8 +726,8 @@ Start.prototype = {
                // verification
                this._isOAuthVerificationSameBrowser()) ||
                this._isOAuthVerificationDifferentBrowser() ||
-               // any URL with oauth in it
-               /oauth/.test(this._window.location.href);
+               // any URL with 'oauth' in the path.
+               /oauth/.test(this._window.location.pathname);
```

Comment:
This was because my test environment is at https://oauth-sync.dev.lcip.org, and the previous check did not like having "oauth" in the hostname.